pkg/autoscaler/aggregation/max/timewindow_test.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package max_test
```

Comment:
Personal habit I guess :) on a lot of CF ecosystem projects we pretty much always use _test package because we like tests to reflect consumer of package (e.g. makes it easier to spot if your external interface stutters), and because it forces you to avoid tests using private interfaces, which is generally good. Will change it though to be more consistent with this codebase.